enum.c

```diff
@@ -3369,13 +3370,19 @@ drop_i(RB_BLOCK_CALL_FUNC_ARGLIST(i, args))
 
 /*
  *  call-seq:
- *     enum.drop(n)               -> array
+ *    drop(n) -> array
+ *
+ *  For positive integer +n+, returns the last +n+ elements:
```

Comment:
Incorrect.  It drops the first n elements and returns an array of the remaining elements.  You  should have examples with n being 0, 1, and 3 instead of 2, so that the difference is clear.

Reply:
Done.